ipa-server/.applier/group_vars/seed-hosts.yml

```diff
@@ -29,6 +29,11 @@ openshift_cluster_content:
       NAMESPACE_DISPLAY_NAME: "{{ namespace }}"
     tags:
       - ipa_project
+  - name: Deploy MachineConfig to configure SEbool
+    file: "{{ inventory_dir }}/../.openshift/files/01-worker-sebool.yml"
+    tags:
+      - ipa_ocp4
+      - ipa_machineconfig
```

Comment:
could we use the `never` tag here? This way the installation instructions can be simplified such that the second step (deployment) doesn't need any tags (include or exclude).